lightyear/src/connection/netcode/server.rs

```diff
@@ -384,6 +385,7 @@ pub struct NetcodeServer<Ctx = ()> {
     conn_cache: ConnectionCache,
     token_entries: TokenEntries,
     cfg: ServerConfig<Ctx>,
+    pub client_errors: Vec<ConnectionError>,
```

Comment:
Do we need to make this pub?

Reply:
it's accessed outside of `netcode`; do you want an accessor method instead?

Reply:
It looks like the client_errors is only used for the `receive` side of things.
Maybe the `try_update` function of the `NetServer` trait could be updated to something like:
```
fn try_update(&mut self, delta_ms: f64) -> Result<Vec<ConnectionError>, ConnectionError>;
```

(the first connection-errors are those that come from connections, the second one is from errors in `try_update` itself.
Or maybe just
```
fn try_update(&mut self, delta_ms: f64) -> Vec<ConnectionError>;
```
if it's too confusing?

Then there's a bit less dancing around and we can just handle the errors directly after update time